doc/source/user-management.md

```diff
@@ -3,6 +3,51 @@
 This section describes management of users and their
 permissions on JupyterHub.
 
+## Culling user pods
+JupyterHub will automatically delete any user pods that have no activity
+for a period of time. This helps free up computational resources and keeps
+costs down if you are using an autoscaling cluster.
+When these users navigate back to your JupyterHub, they will
+have to start their server again, and the state of their previous session
+will be lost. This is known as "culling".
+
+```eval_rst
+
+.. note::
```

Comment:
Is there a reason this is a ..note rather than just a paragraph?

Reply:
to me the paragraph about inactivity is *related* to culling but kind of breaks the flow of the culling configuration instructions. It feels like an aside, which is why I made it a note.